flexmeasures/data/models/charts/belief_charts.py

```diff
@@ -72,8 +72,13 @@ def chart_for_multiple_sensors(
     **override_chart_specs: dict,
 ):
     sensors_specs = []
-    min_resolution_in_ms = (
-        min(sensor.event_resolution for sensor in sensors).total_seconds() * 1000
+    minimum_non_zero_resolution_in_ms = (
+        min(
+            sensor.event_resolution
+            for sensor in sensors
+            if sensor.event_resolution > timedelta(0)
+        ).total_seconds()
```

Comment:
I believe this would break if all sensors record instantaneous values.

Reply:
Great catch!